src/main.ts

```diff
@@ -168,6 +168,12 @@ async function run(): Promise<void> {
         resource: eventTriggerResource,
         service: eventTriggerService,
       };
+
+      if (eventTriggerRetry) {
+        cf.eventTrigger.failurePolicy = {
+          retry: {},
```

Comment:
Don't we need to specify something here if `eventTriggerRetry`?

Reply:
No, and it's dumb. `retry` is a `oneOf` with zero values. This is how you "enable" it. The default behavior is to omit the `failurePolicy` entirely (and thus not enable retries).

Reply:
Ah gotcha, maybe lets add a comment explaining this? LGTM otherwise!